source/features/download-folder-button.tsx

```diff
@@ -1,36 +1,18 @@
 import React from 'dom-chef';
-import select from 'select-dom';
 import {DownloadIcon} from '@primer/octicons-react';
 import * as pageDetect from 'github-url-detection';
 
 import features from '../feature-manager';
-import {getRepo} from '../github-helpers';
+import observe from '../helpers/selector-observer';
 
-function getDropdownItem(downloadUrl: URL): JSX.Element {
```

Comment:
This is no longer practical. There are no icons, classes, titles, aria-label, nothing in the dropdown. It's completely unselectable. This is the kind of issues we'll find more and more as GitHub switches more of its interface to React and CSS-in-JS random classes. 

Here the solution is to just move the button back out of the dropdown. 

Simplification of the features is key.